.github/CONTRIBUTING.md

```diff
@@ -31,70 +31,93 @@ If you at any point have any questions please read: [I have a question related t
-  - **Update the [cookbook](https://cookbook.kirby.design) with examples and showcases**
-4. **Get your changes reviewed:**
-  - **Create a pull-request** - see [creating pull-requests](#creating-pull-requests) 
-  - **Request that the pull-request is code-reviewed in #kirby-guild** - stay responsive to comments and suggestions that might be submitted.
-  - **Request that the pull-request is [UX reviewed](#ux-review)**
-  - **Get your pull-request merged** - when the pull-request has been approved it will automatically be merged to master via [automerge](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/automatically-merging-a-pull-request).
-5. **Celebrate! 🎉** - your changes have been added to Kirby!
-
-### I have an enhancement 
-- Submit [new issue](https://github.com/kirbydesign/designsystem/issues/new/choose) using the enhancement request template
-  - please read "[the good: issue](#issue)"
-- Contact kirby-guild to get it tech refined 
-- If you would like to implement issue: 
-  -please read: 
-    - "[Setup environment and tooling for contribution](#setup-environment-and-tooling-for-contribution)" to get properly setup 
-    - "[Creating pull-requests](#creating-pull-requests)" to get informed on how to most easily get your changes merged
-  - assign yourself to the issue to signal to others you are working on it
-  - if you at any point have questions while solving the issue please see: [I have a question related to Kirby](#i-have-a-question-related-to-kirby)
-
-### I would like to report a bug 
-- Submit [new issue](https://github.com/kirbydesign/designsystem/issues/new/choose) using the bug template  
-  - please read "[the good: issue](#issue)"
-- Contact kirby-guild to get it verified 
-  - we verify because it might be a feature not a bug 🐛
-- If you would like to implement issue: 
-  -please read: 
-    - "[Setup environment and tooling for contribution](#setup-environment-and-tooling-for-contribution)" to get properly setup 
-    - "[Creating pull-requests](#creating-pull-requests)" to get informed on how to most easily get your changes merged
-  - assign yourself to the issue to signal to others you are working on it
-  - if you at any point have questions while solving the issue please see: [I have a question related to Kirby](#i-have-a-question-related-to-kirby)
+1. **Open a [new issue](https://github.com/kirbydesign/designsystem/issues/new/choose) using the "component request"-template** (See guidelines on how to write [the good issue](#issue)).
+2. **Refinement:**
+    - Request that the issue is [UX refined](#ux-refinement) in [#kirby-guild](#getting-access-to-the-kirby-guild-slack-channel); do not proceed until this is done.
+    - Request that the issue is [tech refined](#tech-refinement) in [#kirby-guild](#getting-access-to-the-kirby-guild-slack-channel); do not proceed until this is done.
```

Comment:
In addition to other comment about capitalization.

This is an example where we should *not* use capitalization, because the link text (*tech refined*) is *not* the heading of the section we're linking to (*Tech refinement*)